api/envoy/extensions/transport_socket/capture/v2/capture.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+
+// Trace capture format for the capture transport socket extension. This dumps plain text read/write
+// sequences on a socket.
+
+package envoy.extensions.transport_socket.capture.v2;
```

Comment:
Same comment here about v2alpha?

Additionally, I would suggest we start an entirely new namespace for sharing the outputs of tap/dump stuff. Although the stuff I will be working on (HTTP filter) is going to be different, there is going to be overlap and it would be good to share concepts.

extensions/common/tap/v2alpha? (Or pick something you like better along those lines.)